### ISSUE TITLE
Metadata value issue

### ISSUE THREAD

**Issue by moullos** [mitre-attack/attack-navigator#153]
Hi,

I am trying to import a json file to the beta version of the navigator. The file is being imported successfully and the metadata name fields for the techniques are being shown as expected. However, all the values are being presented as {{metadata.value:

![image](https://user-images.githubusercontent.com/25383165/80113798-70583a00-858b-11ea-8448-18b95b702b50.png)

The metadata field in the json contains an array of metadata objects as defined in [LayerFormatv3](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md). I have been using the same format for the previous versions of the navigator so I am not sure whether I am doing something wrong. Below is a copy of the json leading to the image:
```
{ 
      "techniqueID": "T1106",
      "color": "#f4a261",
      "enabled": true,
      "score": 0.0,
      "metadata": [
        {
          "name": "-WDAG Enabled",
          "value": "False"
        },
        {
          "name": "-AppLocker Enabled",
          "value": "TestNotImplemented"
        },
        {
          "name": "-Software Restriction Policies",
          "value": "TestNotImplemented"
        },
        {
          "name": "-WDEG Enabled",
          "value": "False"
        }
      ]
}
```
Finally, thank you for this awesome tool :)

**Comment by isaisabel** [comment]
Hi @moullos,

Thank for for pointing this out! This seems to be caused by [a typo in the tooltip template](https://github.com/mitre-attack/attack-navigator/blob/develop/nav-app/src/app/matrix/technique-cell/tooltip/tooltip.component.html#L22). We'll make sure to patch it in our next release.

**Comment by isaisabel** [comment]
This bug has been fixed in the [subtechniques-3.1](https://github.com/mitre-attack/attack-navigator/tree/feature/subtechniques-3.1) branch, which is the staging branch for [our next release](https://github.com/mitre-attack/attack-navigator/milestone/4).
